webapp/components/product-info.html

```diff
@@ -0,0 +1,126 @@
+<dom-module id="browser-info">
```

Comment:
Browser info is just migrated code from `test-run` and `test-run-query` below, for re-use.